### ISSUE TITLE
CSS FORM INPUT

### ISSUE THREAD

**Issue by francisneedham** [wkhtmltopdf/wkhtmltopdf#2509]
Hi,
can't get rid of my input fields borders with the --enable-forms option enabled.

i need to do some basic styling ...

thanks !

**Comment by joeldenning** [comment]
Does anyone know if controlling the styling of pdf editable fields is possible? I'd love this feature.

**Comment by blittle** [comment]
@ashkulz is this something that you would need help working on?

**Comment by mtaracena-conexio** [comment]
Get the bytes, search the following string "/BS<</S/I>>" and remove it from the pdf byte Array. 

you can do it manually first, open the pdf with notepad++ search for that string and delete it, save and check if the borders are gone.

**Comment by bluespidernav** [comment]
Yes that's hardly an appealing solution now is it. I would agree after having tried to remove the border by use of appropriate CSS that it does not seem possible. The suggested solution of filtering the output PDF particularly for a large document where perhaps only one of many input html documents exhibits the issue is a complete and total no brainer. Unfortunately the only option appears to be to remove and replace <input> tags with something that obviously won't work with existing form layouts and label_for specifically for the purpose of PDF report generation. Or maybe choose one of the many many other feature limited PDF conversion options that will give you a rich set of new issues.
